automation/src/test/scala/org/broadinstitute/dsde/test/api/AuthDomainMultiGroupApiSpec.scala

```diff
@@ -222,19 +216,14 @@ class AuthDomainMultiGroupApiSpec extends AnyFreeSpec with Matchers with Workspa
             "can be seen but is not accessible" in {
 
               val user = UserPool.chooseProjectOwner
-              implicit val authToken: AuthToken = authTokenDefault
 
               withGroup("AuthDomainOne") { groupOne =>
-                withGroup("AuthDomainTwo", List(user.email)) { groupTwo =>
```

Comment:
Should there still be an `AuthDomainTwo` group here?

Reply:
It's not used.